app/controllers/steps/client/benefit_check_result_controller.rb

```diff
@@ -10,6 +10,10 @@ def edit
       def update
         update_and_advance(BenefitCheckResultForm, as: :benefit_check_result)
       end
+
+      def additional_permitted_params
+        %i[confirm_benefit_check_result]
```

Comment:
I'm confused about this. Should not be necessary if the attributes are declared in the form object.